airform/api/index.js

```diff
@@ -1,6 +1,6 @@
 const { withUiHook } = require('@zeit/integration-utils')
-const formBuilder = require('./form-builder')
-const deploy = require('./deploy')
+const formBuilder = require('_utils/form-builder')
+const deploy = require('_utils/deploy')
```

Comment:
```suggestion
const formBuilder = require('./_utils/form-builder')
const deploy = require('./_utils/deploy')
```
I think it has to be relative paths?